templates/_templating_exclude.json

```diff
@@ -0,0 +1,13 @@
+{
+  "iris" : [],
+  "iris-grib" : [],
+  "tephi" : [
+    "index.ipynb"
+  ],
+  "python-stratify" : [],
+  "iris-esmf-regrid":  [],
+  "cf-units" : [],
+  "nc-time-axis" : [],
+  "workflows" : [],
+  "test-iris-imagehash" : []
+}
```

Comment:
Need to add more templates in here

Reply:
https://github.com/SciTools/.github/pull/72/commits/4ae1b2961733ba093deca2e07e857d8a3a28b6c5